src/usr/lib/rsetup/cli/baota.sh

```diff
@@ -0,0 +1,11 @@
+# shellcheck shell=bash
+
+install_baota() {
+    __parameter_count_check 0 "$@"
+    curl -s https://download.bt.cn/install/install-ubuntu_6.0.sh | sudo bash /dev/stdin ed8484bec
+}
+
+uninstall_baota() {
+    __parameter_count_check 0 "$@"
+    curl -s https://download.bt.cn/install/bt-uninstall.sh | sudo bash /dev/stdin
```

Comment:
```suggestion
    sudo bash < <(curl -s "https://download.bt.cn/install/bt-uninstall.sh")
```
Just to look a little bit nicer.